server/src/resolvers/getVariantsResolver/utils/annotateGnomad.ts

```diff
@@ -1,67 +1,87 @@
-import GnomadAnnotationModel from '../../../models/GnomadAnnotationModel';
+import { GnomadGRCh37AnnotationModel, GnomadGRCh38AnnotationModels } from '../../../models';
 import getCoordinates from '../../../models/utils/getCoordinates';
-import { GnomadAnnotation, GnomadAnnotations, VariantQueryDataResult } from '../../../types';
+import {
+  GnomadAnnotations,
+  GnomadGenomeAnnotation,
+  GnomadGRCh37ExomeAnnotation,
+  VariantQueryDataResult,
+} from '../../../types';
+import resolveAssembly from './resolveAssembly';
+import resolveChromosome from './resolveChromosome';
 import { timeitAsync } from '../../../utils/timeit';
 
 const annotateGnomad = timeitAsync('annotateGnomad')(
-  async (queryResponse: VariantQueryDataResult[]): Promise<VariantQueryDataResult[]> => {
+  async (
+    assemblyId: string,
+    position: string,
+    queryResponse: VariantQueryDataResult[]
+  ): Promise<VariantQueryDataResult[]> => {
+    const resolvedAssemblyId = resolveAssembly(assemblyId);
+    const { chromosome } = resolveChromosome(position);
     const annotationCoordinates = getCoordinates(queryResponse);
+    const GnomadAnnotationModel =
+      resolvedAssemblyId === 'GRCh37'
+        ? GnomadGRCh37AnnotationModel
+        : GnomadGRCh38AnnotationModels[chromosome];
 
-    const annotations = await GnomadAnnotationModel.getAnnotations(
-      annotationCoordinates,
-      'gnomAD_GRCh37'
-    );
+    const annotations = await GnomadAnnotationModel.getAnnotations(annotationCoordinates);
 
     return annotate(queryResponse, annotations);
```

Comment:
We could add some error checking here: `if annotations.status === "fulfilled" return annotate(queryResponse....)`

(in a similar way that CADD annotations do error checking in `getVariantsResolver.ts.`